Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles_test.py

```diff
@@ -1089,6 +1089,27 @@ def test_eml_contains_htm_attachment_empty_file(mocker):
     assert results[0]['EntryContext']['Email'][0]['AttachmentNames'] == ['unknown_file_name0', 'SomeTest.HTM']
 
 
+def test_eml_contains_attachment_with_unknown_encoded_file_name(mocker):
+    """
+    Given: An email containing an attachment with unknown encoded name.
+    When: Parsing a valid email file with default parameters.
+    Then: The file name is parsed as expected.
+    """
+    mocker.patch.object(demisto, 'args', return_value={'entryid': 'test'})
+    mocker.patch.object(demisto, 'executeCommand',
+                        side_effect=exec_command_for_file('Unknown_encode_attachment_name.eml'))
+    mocker.patch.object(demisto, 'results')
+    # validate our mocks are good
```

Comment:
Why do we need this?

Reply:
This is actually unnecceary (I just copied this section of code from another UT) I deleted it